### PR TITLE
Update bgp global docs

### PR DESCRIPTION
**Pull request by rohitthakur2590**
##### ISSUE TYPE

 
- Docs Pull Request
 

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/modules/iosxr_bgp_global.py [hunk 1]
```diff
@@ -1107,12 +1188,155 @@
 #   default-metric 5
 #  !
 # !
+#
```
Comment (Qalthos) [inline plugins/modules/iosxr_bgp_global.py:1191]: There should be at least some break between examples.
```suggestion

```

File: plugins/modules/iosxr_bgp_global.py [hunk 2]
```diff
@@ -1107,12 +1188,155 @@
...
+#   remote-as 65538
+#   bfd fast-detect strict-mode
+#   bfd multiplier 6
+#   bfd minimum-interval 20
+#  !
+#  vrf vrf1
+#   default-metric 5
+#  !
+# !
+#
```
Comment (Qalthos) [inline plugins/modules/iosxr_bgp_global.py:1335]:
```suggestion

```

File: plugins/modules/iosxr_bgp_global.py [hunk 3]
```diff
@@ -1135,144 +1359,200 @@
...
+#   - vrf vrf2
+#   - neighbor 1.1.1.3
+#   - remote-as 2
+#   - graceful-maintenance
+#   - graceful-maintenance activate inheritance-disable
+#   - graceful-maintenance local-preference 1
+#   - graceful-maintenance as-prepends 2
+#   - bgp enforce-first-as disable
+#   - default-metric 4
 #
```
Comment (Qalthos) [inline plugins/modules/iosxr_bgp_global.py:1553]:
```suggestion

```

File: plugins/modules/iosxr_bgp_global.py [hunk 4]
```diff
@@ -1294,38 +1574,144 @@
...
+#         fast_detect:
+#           strict_mode: true
+#         minimum_interval: 20
+#         multiplier: 6
+#       neighbor_address: 192.0.2.14
+#       remote_as: 65538
 #     socket:
 #       receive_buffer_size: 514
 #       send_buffer_size: 4098
+#
```
Comment (Qalthos) [inline plugins/modules/iosxr_bgp_global.py:1611]:
```suggestion

```

**Review by Qalthos — CHANGES_REQUESTED** [review]
Probably also empty lines before and after the tasks themselves instead of a commented line.

**Review by ashwini-mhatre — APPROVED** [review]

**Pull request merged**